docs/lib/auth/fragments/native_common/signin_web_ui/common.md

```diff
@@ -7,6 +7,12 @@
 
 ## Configure Auth Category
 
+<amplify-callout>
+
+This library currently supports the [Authorization Code Grant](https://aws.amazon.com/blogs/mobile/understanding-amazon-cognito-user-pool-oauth-2-0-grants/) OAuth Flow.
```

Comment:
```suggestion
`AWSCognitoAuthPlugin` currently supports the [Authorization Code Grant](https://aws.amazon.com/blogs/mobile/understanding-amazon-cognito-user-pool-oauth-2-0-grants/) OAuth Flow.
```

Reply:
@royjit Since this is for all native libraries, perhaps instead of `AWSCognitoAuthPlugin` we could say 'this library's Cognito plugin'?

Reply:
yea, that works. Just to avoid confusion in case other auth plugins are added.